src/invidious/yt_backend/youtube_api.cr

```diff
@@ -114,6 +115,12 @@ module YoutubeAPI
       os_version:          ANDROID_VERSION,
       platform:            "MOBILE",
     },
+    ClientType::AndroidTestSuite => {
```

Comment:
According to https://github.com/zerodytrash/YouTube-Internal-Clients, you should send android_sdk_version too
```
private ANDROID_TEST_SUITE_APP_VERSION = "1.9"
private ANDROID_TEST_SUITE_USER_AGENT  = "com.google.android.youtube/1.9 (Linux; U; Android 12; US) gzip"
ClientType::AndroidTestSuite => {
      name:                "ANDROID_TESTSUITE",
      name_proto:          "30",
      version:             ANDROID_TEST_SUITE_APP_VERSION,
      api_key:             "AIzaSyA8eiZmM1FaDVjRy-df2KTyQ_vz_yYM39w",
      android_sdk_version: ANDROID_SDK_VERSION,
      user_agent:          ANDROID_TEST_SUITE_USER_AGENT,
      os_name:             "Android",
      os_version:          ANDROID_VERSION,
      platform:            "MOBILE",
    },
```

Reply:
Interesting. I've been running this PR's code for 24 hours on the test instance, without an issue. I'll add that anyway.